packages/sync/src/modules/Terms.php

```diff
@@ -11,6 +11,14 @@ function name() {
 		return 'terms';
 	}
 
+	public function get_object_by_id( $object_type, $id ) {
+		if ( $object_type === 'term' && $term = get_term( intval( $id ) ) ) {
```

Comment:
I'd love to not introduce any new assignments in conditions. Can we avoid that?

Reply:
Nice, done!